bin/trigger_rc_tests.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+"""
+This script is used to run test_upgrade script.
+"""
+
+import argparse
+import http.client
+import json
+import os
+import re
+import time
+from pathlib import Path
+
+CHECK_PIPELINE_STATUES_TIMER_MIN = 5
+GITHUB_ORG = "astronomer"
+CIRCLECI_URL = "circleci.com"
+REPO = "terraform-aws-astronomer"
+REPO_BRANCH = "master"
+
+parent_directory = Path(__file__).parent.parent
+circleci_directory = parent_directory / ".circleci"
+
+
+def run_workflow(circleci_token: str, parameters: dict = None):
+    circle_ci_conn = http.client.HTTPSConnection(CIRCLECI_URL, timeout=15)
+    api_endpoint = f"/api/v2/project/github/{GITHUB_ORG}/{REPO}/pipeline"
+
+    headers = {"content-type": "application/json", "Circle-Token": circleci_token}
+
+    payload = {"branch": REPO_BRANCH}
+
+    if parameters is not None:
+        payload["parameters"] = parameters
+
+    circle_ci_conn.request(
+        method="POST", url=api_endpoint, body=json.dumps(payload), headers=headers
+    )
+    resp = circle_ci_conn.getresponse().read().decode("utf-8")
+    circle_ci_conn.close()
+    return resp
+
+
+def get_job_state(circleci_token: str, pipeline_id: str):
+    circle_ci_conn = http.client.HTTPSConnection(CIRCLECI_URL)
+    api_endpoint = f"/api/v2/pipeline/{pipeline_id}/workflow"
+
+    headers = {"content-type": "application/json", "Circle-Token": circleci_token}
+
+    circle_ci_conn.request(method="GET", url=api_endpoint, headers=headers)
+    resp = circle_ci_conn.getresponse().read().decode("utf-8")
+    circle_ci_conn.close()
+    return resp
+
+
+def main(circleci_token: str, astro_path: str):
+    # Getting Astronomer Helm Chart - FileName
+    file_list = os.listdir(astro_path)
+
+    astro_version = None
+    for file_name in file_list:
+        x = re.search("astronomer-.*.tgz", file_name)
+        if x is not None and astro_version is None and "rc" in file_name:
+            print(f"INFO: Found file {file_name}")
+            astro_version = file_name
+
+    if astro_version is None:
+        print(
+            f"INFO: Skipping calling workflow as no valid version found for the RC test. Below files are found at path: {astro_path}."
+        )
+        print(json.dumps(file_list))
+        exit(0)
```

Comment:
`exit()` isn't always available.

```suggestion
        raise SystemExit(0)
```